### PR TITLE
Added sramcompiler to mdf format

### PR DESCRIPTION
**Pull request by azidar**

### CODE REVIEW

**Review by colinschmidt — COMMENTED** [review]
This seems straight-forward but I'm not expert.

File: scalalib/src/main/scala/MacroLib.scala [hunk 1]
```diff
@@ -102,39 +102,168 @@ case class SRAMMacro(
...
   }
 }
 
+// SRAM compiler
+case class SRAMGroup(
+  name: Seq[String],
+  family: String,
+  vt: Seq[String],
+  mux: Int,
+  depth: Range,
```
Comment (edwardcwang) [inline scalalib/src/main/scala/MacroLib.scala:152]: Similar concern as in the other PR

**Review by edwardcwang — APPROVED** [review]
Looks good overall except for one concern

File: scalalib/src/main/scala/MacroLib.scala (hunk 1 above)
Comment (azidar) [reply scalalib/src/main/scala/MacroLib.scala:152]: Ok, after talking to Colin, we will make it inclusive on the maximum value.

**Pull request merged**